docs/user_guide/advanced_config.rst

```diff
@@ -104,26 +104,10 @@ low-latency solid state drives.
 Logging
 -------
 
-To simplify debugging, enable logging for the logger named 'datareservoirio'.
+To simplify debugging, enable logging for the logger named :py:mod:`datareservoirio`. This is especially helpful if you experience undesired behavior in your application. 
```

Comment:
on this line we don't refer to the module but the logger. So here I believe we should skip pymod